src/amo/components/PermissionsCard/styles.scss

```diff
@@ -0,0 +1,29 @@
+.PermissionCard-learn-more {
+  margin: 12px auto 0;
```

Comment:
I see a top margin of `28px` in the mocks

Reply:
The 28 I see if between the top of the button and the bottom of the text, but that doesn't take the bottom margin of the `<ul>` into account, so it looks too big to me. If I make it 16 it works with the bottom margin to give us an effective 28.